.github/workflows/build.yml

```diff
@@ -17,14 +17,18 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v4
-      - name: Install ninja
-        uses: seanmiddleditch/gha-setup-ninja@master
-      - name: Install nasm
-        uses: ilammy/setup-nasm@v1
+      - name: Install latest ninja and cmake
+        uses: lukka/get-cmake@latest
```

Comment:
The motivation here is to speed up clang-tidy a bit by not installing nasm, since it shouldn't affect anything clang-tidy is checking. I can add it back if we still need it for linting though.

Reply:
Oh right, of course, this is only for linting. That's fine.